src/js/Dialog.js

```diff
@@ -301,7 +301,7 @@ _context.invoke('Nittro.Extras.Dialogs', function(DOM, CSSTransitions, Arrays, R
 
         _saveFocusedElement: function () {
             this._.origFocusTarget = document.activeElement;
-            this._.origFocusTarget && this._.origFocusTarget.blur();
+            this._.origFocusTarget && this._.origFocusTarget.prototype.blur !== 'undefined' && this._.origFocusTarget.blur();
```

Comment:
this would achieve the same and is a little briefer:
```suggestion
            this._.origFocusTarget && this._.origFocusTarget.blur && this._.origFocusTarget.blur();
```